src/domain/QuestionLists/MoreIconContent.tsx

```diff
@@ -26,7 +28,7 @@ export default function MoreIconContent({
         </Button>
       </Menu.Item>
       <Menu.Item>
-        <Button type="link" danger>
+        <Button type="link" onClick={deleteCallback} danger>
```

Comment:
Just a thought that's not clear to me from the code, will there be a confirmation dialog before it's deleted? 🤔

Reply:
The callback is being received from parent, where the actual delete is being handled and currently is done as expected as far as I can tell from the code (a modal pops up asking whether to actually delete)

Reply:
Delete menu item is connected with delete operation, but a modal window is connected with deleted item because there is some content about deleted item (type, title) and it can be different for multiple items. From my point of view it is better to keep logic of modal window in specific component.